src/components/inputs/TextField.js

```diff
@@ -351,7 +347,7 @@ export default class TextField extends BaseInput {
               }),
               color: floatingPlaceholderState.interpolate({
                 inputRange: [0, 1],
-                outputRange: [placeholderColor, this.getStateColor(floatingPlaceholderColor)]
+                outputRange: [placeholderColor, this.getStateColor(floatingPlaceholderColor || {focus: Colors.blue30, default: DEFAULT_PLACEHOLDER_COLOR})]
```

Comment:
I don't understand, why DEFAULT_PLACEHOLDER_COLOR is not an enum with states if you have 'focus' and 'default' cases?

Reply:
done